R/ResultFlagsIndependent.R

```diff
@@ -516,6 +549,13 @@ BelowNationalWQXLowerThreshold <- function(.data, clean = TRUE) {
 #' dataframe.
 #'
 #' @export
+#' 
+#' @examples 
```

Comment:
Sure, will do! The description says that when clean = FALSE and cleanNA = FALSE, no data is removed from the dataframe and nothing changes. Just want to confirm that is what you want the example to do.

Reply:
Good point, after looking at it again, let's leave this one out of the examples since it doesn't do anything. We should also add a warning note to this function for this scenario when clean = FALSE and cleanNA = FALSE. Would you be interested in adding that to this function?

Reply:
Sure! I can do that.